crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -150,6 +157,48 @@ async fn invalid_nonce_failure_causes_tx_resubmission_under_different_nonce() {
     .expect("valid nonce guard failed");
 }
 
+#[tokio::test]
+async fn test_single_tx_integrity() {
+    let test_composer = spawn_composer(&["test1"]).await;
+
+    // blockhash, blocknumber and transactionindex are generated after execution - we can leave them
+    // out
+    let txs = r#"{
```

Comment:
where is this serialized tx from?

Reply:
Good question - I took this from Derek's PR.

Do you have a better suggestion for what to use?